.gitignore

```diff
@@ -9,12 +9,18 @@ pynest/pynestkernel.cpp
 *.la
 .*
 *.log
+<<<<<<< HEAD
+*.pyc
+config.status
+=======
+>>>>>>> master
 Makefile
```

Comment:
These are more artefacts from an unresolved merge conflict. Please remove them and also the line with `config.status`.